cilium-health/launch/endpoint.go

```diff
@@ -82,70 +79,69 @@ func configureHealthRouting(netns, dev string, addressing *models.NodeAddressing
 	if option.Config.EnableIPv6 {
 		v6Routes, err := connector.IPv6Routes(addressing, mtuConfig.GetRouteMTU())
 		if err != nil {
-			return fmt.Errorf("Failed to get IPv6 routes")
+			return nil, fmt.Errorf("Failed to get IPv6 routes")
 		}
 		routes = append(routes, v6Routes...)
 	}
 
-	prog := "ip"
-	args := []string{"netns", "exec", netns, "bash", "-c"}
-	routeCmds := []string{}
+	return routes, nil
+}
+
+// configureHealthRouting is meant to be run inside the health service netns
+func configureHealthRouting(routes []route.Route, dev string) error {
 	for _, rt := range routes {
-		cmd := strings.Join(rt.ToIPCommand(dev), " ")
-		log.WithField("netns", netns).WithField("command", cmd).Debug("Adding route")
-		routeCmds = append(routeCmds, cmd)
+		cmd := rt.ToIPCommand(dev)
+		prog := cmd[0]
+		args := cmd[1:]
+		log.Debugf("Running \"%s %+v\"", prog, args)
+		out, err := exec.Command(prog, args...).CombinedOutput()
```

Comment:
Would we want to restrict warning on this to just stderr?

Reply:
Maybe - I left it like it was previously tho.